locations/spiders/petsathome_gb.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
+import scrapy
+import json
+import csv
+from locations.linked_data_parser import LinkedDataParser
+from scrapy.http import FormRequest
+
+
+# This method could be moved into library code and documented, tested etc, supports calls of the form:
+# get_locations("eu_centroids_40km_radius_country.csv")
+# get_locations("eu_centroids_40km_radius_country.csv", ["GB", "IE"])
+# get_locations("us_centroids_50mile_radius_state.csv", "NY")
+def get_locations(areas_csv_file, area_field_filter=None):
```

Comment:
In spider this seems like overkill, but if it ends up shared code, I can understand it,